### PR TITLE
EVA Website

### PR DESCRIPTION
**Pull request by EDGAhab**

### CODE REVIEW

**Comment by gaurav274** [comment]
@EDGAhab Why do we have the entire eva repository inside the website?
And is it possible to directly use the Readme on GitHub rather than making a copy inside the website folder? If not, please update the readme. Thanks!

**Comment by jarulraj** [comment]
Can we get rid of the eva source code?

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+logo: Logo.png
+
+title: EVA
+blank: 
+description: Visual data management system. Features include declarative query language, storage management, deep learning, and hardware acceleration. Developed at Georgia Institute of Technology.
+
+baseurl: "/eva" # the subpath of your site, e.g. /blog
+url: "https://georgia-tech-db.github.io" # the base hostname & protocol for your site, e.g. http://example.com
+
+show_downloads: true
```
Comment (jarulraj) [inline _config.yml:29]: We can disable this option.

**Comment by EDGAhab** [comment]
> Can we get rid of the eva source code?

I just got rid of the source code!

**Comment by EDGAhab** [comment]
> @EDGAhab Why do we have the entire eva repository inside the website? And is it possible to directly use the Readme on GitHub rather than making a copy inside the website folder? If not, please update the readme. Thanks!

I deleted the eva repo and updated the readme.md!

File: _config.yml (hunk 1 above)
Comment (EDGAhab) [reply _config.yml]: Okay! Disabled

**Pull request closed without merging**